src/HttpClient.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Http\Client;
+
+use Psr\Http\Message\StreamInterface;
+use Psr\Http\Message\UriInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Interface for HTTP conventional methods
```

Comment:
Hm. Sounds good, but does it really explain what this interface is about? The point is rather about HTTP methods than the lack of RequestInterface.